bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/internal/ThingManagerImpl.java

```diff
@@ -623,7 +624,7 @@ public void thingUpdated(final Thing thing, ThingTrackerEvent thingTrackerEvent)
                                 setThingStatus(thing,
                                         buildStatusInfo(ThingStatus.UNINITIALIZED,
                                                 ThingStatusDetail.HANDLER_CONFIGURATION_PENDING,
-                                                "@text/missing-or-invalid-configuration"));
+                                                e.getValidationMessages().toString()));
```

Comment:
Do you thing it is possible to get the localized messages here by calling [getValidationMessages(Locale)](https://github.com/openhab/openhab-core/blob/cabb3f73150d13f44d647f46af8362e396684b92/bundles/org.openhab.core.config.core/src/main/java/org/openhab/core/config/core/validation/ConfigValidationException.java#L92)?